### ISSUE TITLE
2.4.0 throws an error

### ISSUE THREAD

**Issue by goooseman** [TypedProject/ts-gphoto2-driver#20]
### Steps to reproduce the issue:
1. Install 2.4.0 version of the library
2. Use it

### Actual
![Screen Shot 2020-05-06 at 6 54 02](https://user-images.githubusercontent.com/3503252/81137255-d5863500-8f66-11ea-80f0-16bd15249084.png)
![Screen Shot 2020-05-06 at 6 57 32](https://user-images.githubusercontent.com/3503252/81137260-dfa83380-8f66-11ea-85d3-39e137f6eb75.png)

TS compiler leaves 
```
require("./@types/ref-array-napi");
require("./@types/ref-struct-napi");
```

in the JS code, which can not be imported.

Luckily, this problem is already fixed in the production branch. Just need to do a patch release.

**Comment by Romakita** [comment]
I have seen that the latest commit hasn't message with fix or feat. So the build haven't produce a new npm package.
I changed the latest commit message. The package will be release in few minutes :)

**Comment by goooseman** [comment]
I confirm that new version is released and working with NodeJS v12.13.0 in Electron v8.2.5 project.

**Comment by Romakita** [comment]
Coool :D
